tests/crypto/CMakeLists.txt

```diff
@@ -42,7 +42,13 @@ add_executable(cncrypto-tests
   ${crypto_headers})
 target_link_libraries(cncrypto-tests
   PRIVATE
+    wallet
+    cryptonote_core
     common
+    blockchain_db
+    p2p
+    epee
+    device
```

Comment:
o_O Surely this is not needed, right ?

Reply:
The linker make me crazy about think found or not. YES I need help on this point. Could someone check and try to clean the dependencies?